toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -8,7 +8,7 @@ bash-4.4.18-7.cm2.aarch64.rpm
 bash-debuginfo-4.4.18-7.cm2.aarch64.rpm
```

Comment:
Please update mariner-rpm-macros release number in these 4 files as well.

Reply:
I think this is resolved